tests/topotests/ldp-oc-acl-topo1/r1/ldpd.conf

```diff
@@ -0,0 +1,25 @@
+hostname r1
```

Comment:
why are we committing __init__.py into topotests?

Reply:
Removing init.py.

(I saw __init__.py checked into other topotest directories, so I thought this was the practice.   My mistake.)